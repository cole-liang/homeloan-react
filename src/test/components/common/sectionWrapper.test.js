import React from "react";
import { shallow } from "enzyme";
import SectionWrapper from "../../../components/common/sectionWrapper";

describe("SectionWrapper Component Test", () => {
  const children = <div>Test message</div>;
  const name = "Test Section";
  const props = { children, name };

  it("should render SectionWrapper component", () => {
    const wrapper = shallow(<SectionWrapper {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
