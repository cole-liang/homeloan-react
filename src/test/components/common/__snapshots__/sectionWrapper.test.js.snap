// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionWrapper Component Test should render SectionWrapper component 1`] = `
<styled.div>
  <span
    className="wrapperName"
  >
    Test Section
  </span>
  <div
    className="wrapper"
  >
    <div>
      Test message
    </div>
  </div>
</styled.div>
`;
